frontend/src/components/templates/InstructorEnrollmentList.tsx

```diff
@@ -183,7 +183,7 @@ const InstructorEnrollmentList: React.FC = () => {
         enableSorting: false,
         filterFn: 'includesString',
         meta: {
-          placeholder: 'Filtrer parmi tous les emails de contact',
+          placeholder: 'ex : mairie@paris.fr',
```

Comment:
Fonctional issue here : the search is amongst all contact email not only the demandeur email (as displayed in the header of the column).

Reply:
Nonetheless, as it's an improvment we should merge this and submit the issue to @SchweisguthN

Reply:
Agreed, and I'll add a ticket to write a correct title later